libs/remix-ui/run-tab/src/lib/actions/deploy.ts

```diff
@@ -64,10 +64,19 @@ export const getSelectedContract = (contractName: string, compiler: CompilerAbst
 
       return txHelper.inputParametersDeclarationToString(constructorInteface.inputs)
     },
-    isOverSizeLimit: () => {
-      const deployedBytecode = contract.object.evm.deployedBytecode
+    isOverSizeLimit: async (args: string) => {
+      const encodedParams = await txFormat.encodeParams(args, txHelper.getConstructorInterface(contract.object.abi))
+      const bytecode = contract.object.evm.bytecode.object + (encodedParams as any).dataHex
+      // https://eips.ethereum.org/EIPS/eip-3860
+      const initCodeOversize = (bytecode && bytecode.length / 2 > 2 * 24576)
```

Comment:
Can you separate condition with brackets to make it easier to read?

Reply:
I've added the "translation" label, for @ryestew